tools/log-collector/log_collector.go

```diff
@@ -30,16 +32,33 @@ import (
 	"github.com/trilioData/tvk-plugins/internal"
 )
 
+type GroupVersionKind struct {
```

Comment:
add validation for this struct - at least `kind` should be given

Reply:
done